### DISCUSSION TITLE
Release Tunny v0.8.2

### DISCUSSION THREAD

**Discussion by hrntsm** [Tunny-gh/Tunny_in_Open_Sea#195] [Announcements]
## Changed

- Stopped putting built files together in gha.
  - Because some people did not work in some environments
- If the objective function contains null for 10 consecutive times, optimization is stopped.
- Update python lib
  - bump up optuna v3.3 & optuna-dashboard v0.12.0 & some library

## Fixed

- NSGA-III supports constraints, but was getting a message that constraints are not taken into account

## Related PR
* Fix/nsga iii constraint by @hrntsm in https://github.com/hrntsm/Tunny/pull/190
* Fix/fody error by @hrntsm in https://github.com/hrntsm/Tunny/pull/191
* Fix/ten times null exception by @hrntsm in https://github.com/hrntsm/Tunny/pull/192
* Feature/update python lib by @hrntsm in https://github.com/hrntsm/Tunny/pull/193
* Update v0.8.2 by @hrntsm in https://github.com/hrntsm/Tunny/pull/194

**Full Changelog**: https://github.com/hrntsm/Tunny/compare/v0.8.1...v0.8.2

<hr /><em>This discussion was created from the release <a href='https://github.com/hrntsm/Tunny/releases/tag/v0.8.2'>Tunny v0.8.2</a>.</em>

**Comment by netsonicyxf** [comment]
When running the sample: Vase_Shape_opt.gh, it shows error
![image](https://github.com/hrntsm/Tunny/assets/43513391/3382a638-b19e-4a26-b2fe-ce93aad89162)

**Reply by hrntsm** [reply]
This has not yet been corrected because the cause has not been identified
